docs/ovh.md

```diff
@@ -24,12 +24,33 @@
 
 - `"domain"`
 - `"host"` is your host and can be a subdomain or `"@"`
+
+#### Using DynHost
+
 - `"username"`
 - `"password"`
 
+#### OR Using ZoneDNS
+
+- `"api_endpoint"` default is `"ovh-eu"`
+- `"app_key"`
+- `"app_secret"`
+- `"consumer_key"`
+
+The ZoneDNS implementation allows you to update any record name including *.yourdomain.tld
+
 ### Optional parameters
 
 - `"ip_version"` can be `ipv4` (A records) or `ipv6` (AAAA records), defaults to `ipv4 or ipv6`
 - `"provider_ip"` can be set to `true` to let your DNS provider determine your IPv4 address (and/or IPv6 address) automatically when you send an update request, without sending the new IP address detected by the program in the request.
+- `"zone_dns"` select between two modes, OVH's dynamic hosting service (`false`) or OVH's API (`true`). Default is `false`
```

Comment:
Maybe make that a string in case in the future there is another mode? As in, it's unlikely, but it's good practice 😉 Plus it's easier to remember for the user to have `"mode": "dynamic"` or `"mode": "api"` or something like that, than remembering what `true` means.

Reply:
That's a very valid point ^^ I'll implement that